packages/components/pie-button/src/index.ts

```diff
@@ -43,6 +58,95 @@ export class PieButton extends LitElement implements ButtonProps {
     @property({ type: Boolean })
     public isFullWidth = false;
 
+    @property({ type: String })
+    public name?: string;
+
+    @property({ type: String })
+    public value?: string;
+
+    @property()
+    public formaction: ButtonProps['formaction'];
+
+    @property()
+    public formenctype: ButtonProps['formenctype'];
+
+    @property()
+    public formmethod: ButtonProps['formmethod'];
+
+    @property({ type: Boolean })
+    public formnovalidate: ButtonProps['formnovalidate'];
+
+    @property()
+    public formtarget: ButtonProps['formtarget'];
+
+    /**
+     * This method creates an invisible button of the same type as pie-button. It is then clicked, and immediately removed from the DOM.
+     * This is done so that we trigger native form actions, such as submit and reset in the browser. The performance impact of adding and removing a single button to the DOM
+     * should be neglible, however this should be monitored.
+     * This is the only viable way of guaranteeing native button behaviour when using a web component in place of an actual HTML button.
+     *
+     * TODO: if we need to repeat this logic elsewhere, then we should consider moving this code to a shared class or mixin.
+     */
+    private _simulateNativeButtonClick (btnType: 'submit' | 'reset') {
+        if (!this.form) return;
+
+        const btn = document.createElement('button');
+        btn.type = btnType;
+
+        // Visually hidden styles
+        btn.style.position = 'absolute';
+        btn.style.width = '1px';
+        btn.style.height = '1px';
+        btn.style.padding = '0';
+        btn.style.margin = '-1px';
+        btn.style.overflow = 'hidden';
+        btn.style.border = '0';
+        btn.style.whiteSpace = 'nowrap';
```

Comment:
nit: do you think we should create a utility class somewhere in `pie-css` called `visually-hidden` and add to the element instead? 

something like: `btn.classList.add('visually-hidden');`

```
  position: absolute;
  width: 1px;
  height: 1px;
  padding: 0;
  margin: -1px;
  overflow: hidden;
  border: 0;
  white-space: nowrap;
```

also in case it helps, we have a mixin called `hidden-pseudo` in snacks that are used across multiple places that provides a similar thing.

Reply:
@raoufswe I agree - I am going to add a helper class to `pie-css` in an immediate follow up PR :) just want to keep this one minimal 👍